app/models/user.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id              :bigint           not null, primary key
+#  first_name      :string           not null
+#  last_name       :string
+#  email           :string           not null
+#  designation     :string           not null
+#  age             :integer          default(18)
+#  gender          :string           not null
+#  address         :text
+#  twitter_url     :string
+#  linkedin_url    :string
+#  github_url      :string
+#  organisation_id :bigint           not null
+#  role_id         :bigint           not null
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#
+class User < ApplicationRecord
+  belongs_to :organisation
```

Comment:
@apeksha2697  Here, please separate associations and validations by one-line spacing, as well as segregate by adding comments like #associations and #validations

Reply:
done